tasks.py

```diff
@@ -80,14 +80,22 @@ def qa(ctx):
     header(qa.__doc__)
     with ctx.cd(ROOT):
         info('Python Static Analysis')
-        flake8_results = ctx.run('flake8 udata_piwik', pty=True, warn=True)
+        flake8_results = ctx.run('flake8 udata_ckan', pty=True, warn=True)
```

Comment:
copy-paste error ;) Should be `flake8 udata_piwik`

Reply:
Arf thanks. It does not even raise an error!